### PR TITLE
Update graalvm github workflow

### PR DESCRIPTION
**Pull request by radovanradic**

### CODE REVIEW

File: .github/workflows/graalvm.yml [hunk 1]
```diff
@@ -21,6 +21,9 @@ jobs:
       matrix:
         java: ['17']
         graalvm: ['latest', 'dev']
```
Comment (radovanradic) [inline .github/workflows/graalvm.yml:23]: @alvarosanchez Is this the change you suggested?

Comment (alvarosanchez) [reply .github/workflows/graalvm.yml:23]: In theory, yes, but I tried [here](here) and it still generated 3 jobs:

image

I will try with `exclude` instead.

Comment (alvarosanchez) [reply .github/workflows/graalvm.yml:23]: This did the trick:

```
      matrix:
        graalvm: ['latest', 'dev']
        java: ['17']
        include:
          - graalvm: 'dev'
            java: '19'
        exclude:
          - graalvm: 'dev'
            java: '17'
```

**Review by alvarosanchez — CHANGES_REQUESTED** [review]

File: .github/workflows/graalvm.yml (hunk 1 above)
Comment (radovanradic) [reply .github/workflows/graalvm.yml]: Updated

**Review by alvarosanchez — APPROVED** [review]

**Pull request merged**